x-pack/plugins/endpoint/server/services/resolver/common.test.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { parsePhase0EntityID, isPhase0EntityID, buildPhase0EntityID } from './common';
+
+describe('phase0 entity identification and parsing', () => {
```

Comment:
More references to `phase 0` that should probably be changed to something that doesn't require internal knowledge of the project.